src/Workspaces/CSharp/Portable/Classification/Worker_Preprocesser.cs

```diff
@@ -324,6 +328,43 @@ private void ClassifyLoadDirective(LoadDirectiveTriviaSyntax node)
         ClassifyDirectiveTrivia(node);
     }
 
+    private void ClassifyIgnoredDirective(IgnoredDirectiveTriviaSyntax node)
+    {
+        AddClassification(node.HashToken, ClassificationTypeNames.PreprocessorKeyword);
+        AddClassification(node.ColonToken, ClassificationTypeNames.PreprocessorKeyword);
+
+        // The first part (separated by whitespace) of content is a "keyword", e.g., 'sdk' in '#:sdk Test'.
+        if (TryFindWhitespace(node.Content.ValueText, out var firstSpaceIndex))
```

Comment:
```suggestion
        var firstSpaceIndex = node.Content.Text.IndexOf(" ");
```


Don't use 'valueText'.  it is interpreted characters, not the actual characters in the file.

Reply:
Thanks, there is so many of these - ValueText, Text, ToString, ToFullString, ...

Reply:
yup.  never use 'Value' if actually trying to get real positions infiles :)